cmd/webhook/main.go

```diff
@@ -130,9 +144,25 @@ func main() {
 		net.SchemeGroupVersion.WithKind("ServerlessService"):             &net.ServerlessService{},
 	}
 
+	configHandlers := configmap.Constructors{
+		tracingconfig.ConfigName:         tracingconfig.NewTracingConfigFromConfigMap,
+		autoscaler.ConfigName:            autoscaler.NewConfigFromConfigMap,
+		certconfig.CertManagerConfigName: certconfig.NewCertManagerConfigFromConfigMap,
+		// gc.ConfigName:                    gc.NewConfigFromConfigMapFunc(logger, minRevisionTimeout),
```

Comment:
I wonder what the value for `minRevisionTimeout` should be here? Should it be set to the default `10*time.Hour`?

Reply:
Looks like `controller.GetResyncPeriod(ctx)` is what we use in the GC controller.

Reply:
I dont think we set the value for `resyncPeriodKey{}` on webhook's `ctx`, so `controller.GetResyncPeriod(ctx)` always falls back to its default, hence the question above. but sure I just reuse `controller.GetResyncPeriod(ctx)`.  

https://github.com/knative/serving/blob/508ab0f9a4f3c66f3d623ce2a610c155397805e1/vendor/knative.dev/pkg/controller/controller.go#L461-L467